server/taskflows/hpccloud/taskflow/openfoam/DockerOpenFOAM_run.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+cd $1
+
+# Env setup
+source /opt/OpenFOAM/OpenFOAM-v1612+/etc/bashrc
```

Comment:
The /opt/OpenFOAM.... path should probably be cluster configuration parameter for the workflow?